docs/js/extra.js

```diff
@@ -51,3 +51,42 @@ function setCopyText() {
 document$.subscribe(function () {
   setCopyText();
 });
+
+// Use client-side redirects for anchors that have moved.
+// Other redirects should use `redirect_maps` in the `mkdocs.yml` file instead.
+(function () {
```

Comment:
We'll probably need to add an event listener for this, mostly on [`DOMContentLoaded`](https://developer.mozilla.org/en-US/docs/Web/API/Document/DOMContentLoaded_event).

Reply:
Why? I don't think you can get to a page that doesn't exist except on initial load — i.e., like only an external link should reference it.

Reply:
Oh right, because all the internal references have been updated.